src/store/actions/taskActions.jsx

```diff
@@ -0,0 +1,11 @@
+import { ADD_TASK } from "src/store";
+
+export const addTasks = (task) => {
+  task.id = `id-${Math.random().toString(36).substr(2, 9)}-${Date.now()}`;
+  task.is_done = false;
```

Comment:
use camelCase for this type of value.
```suggestion
  task.isCompleted = false;
```